$NetBSD: patch-ac,v 1.2 2000/03/05 17:31:52 kim Exp $

Make jove turn on auto-fill-mode for elm and nn files.

--- doc/jove.rc.orig	Sun Jan 22 09:44:57 1995
+++ doc/jove.rc	Sun Mar  5 19:30:21 2000
@@ -13,7 +13,7 @@
 
 # Set fill mode when Jove is fired up automatically by certain
 # mail/news readers.
-1 auto-execute-command auto-fill-mode /tmp/\{Re\|article\|rnmail\|pn\}
+1 auto-execute-command auto-fill-mode .*/tmp/\{Re\|article\|nn\|rnmail\|pn\|snd\}
 
 # The following is if you prefer fill-mode to be on for all
 # files except the ones you want it off for (but on the other hand it could
